keras/src/backend/numpy/core.py

```diff
@@ -18,6 +18,8 @@
 SUPPORTS_RAGGED_TENSORS = False
 IS_THREAD_SAFE = True
 
+_print = print
```

Comment:
nitpick: instead of doing this, let's use `__builtins__`:

```python
def print(*args, **kwargs):
  return __builtins__.print(*args, **kwargs)
```

Reply:
Oh that's a good one, I didn't know the `__builtins__`